lib/public/IAvatar.php

```diff
@@ -80,7 +81,7 @@ public function remove();
 	/**
 	 * Get the file of the avatar
 	 * @param int $size -1 can be used to not scale the image
-	 * @return File
+	 * @return ISimpleFile|InMemoryFile
```

Comment:
InMemoryFile implements ISimpleFile, so:
```suggestion
	 * @return ISimpleFile
```